_usage/limitations-of-babelfish.md

```diff
@@ -9,89 +9,90 @@ nav_order: 5
+| `ALTER` statement | You can't use an `ALTER` statement to modify the following objects: `AGGREGATE`, `APPLICATION ROLE`, `ASSEMBLY`, `ASYMMETRIC KEY`, `AUTHORIZATION`, `AVAILABILITY GROUP`, `BROKER PRIORITY`, `COLUMN ENCRYPTION KEY`, `CONTRACT`, `BACKUP CERTIFICATE`, `CREDENTIAL`, `TABLE ... IDENTITY`, `USER`, `CRYPTOGRAPHIC PROVIDER`, `DATABASE ENCRYPTION KEY`, `DATABASE AUDIT SPECIFICATION`, `DEFAULT`, `ENDPOINT`, `EXTERNAL FILE FORMAT`, `EVENT NOTIFICATION`, `EVENT SESSION`, `FULLTEXT CATALOG`, `FULLTEXT INDEX`, `FULLTEXT STOPLIST`, `INDEX`, `SPATIAL INDEX`, `XML INDEX`, `COLUMNSTORE INDEX`, `EXTERNAL LANGUAGE`, `EXTERNAL LIBRARY`, `LOGIN`, `MASTER KEY`, `MESSAGE TYPE`, `EXTERNAL LANGUAGE`, `EXTERNAL LIBRARY`, `LOGIN`, `PARTITION FUNCTION`, `PARTITION SCHEME`, `QUEUE`, `REMOTE SERVICE BINDING`, `RESOURCE POOL`, `EXTERNAL RESOURCE POOL`, `RESOURCE GOVERNOR`, `ROLE`, `ROUTE`, `RULE`, `SCHEMA`, `SEARCH PROPERTY LIST`, `SECURITY POLICY`, `SEARCH PROPERTY LIST`, `SERVER AUDIT`, `SERVER AUDIT SPECIFICATION`, `SERVER ROLE`, `SERVICE`, `SERVICE MASTER KEY`, `SYMMETRIC KEY`, `TABLE ... GRANT IDENTITY` clauses, `EXTERNAL TABLE`, `TRIGGER` (schema qualified), `TYPE`, `USER`, `WORKLOAD GROUP`, `WORKLOAD CLASSIFIER`, `SELECTIVE XML INDEX`, `XML SCHEMA COLLECTION` |
 | `CREATE/ALTER/DROP APPLICATION ROLE` | Functionality related to this syntax is not supported. |
 | `CREATE/ALTER/DROP ASSEMBLY` | Functionality related to this syntax is not supported. |
 | `CREATE/ALTER/DROP ASYMMETRIC KEY` | Functionality related to this syntax is not supported. |
 | Assembly modules and CLR routines | Functionality related to assembly modules and CLR routines is not supported. |
 | `CREATE/ALTER/DROP AUTHORIZATION` | Functionality related to these commands is not supported. |
 | `CREATE/ALTER/DROP AVAILABILITY GROUP` | Functionality related to these commands is not supported. |
 | `BACKUP` statement | PostgreSQL snapshots of a database are dissimilar to backup files created in SQL Server. Also, there may be differences between SQL Server and PostgreSQL in the granularity of when a backup and restore occurs. |
```

Comment:
The issue is simpler than this: we do not support restoring of MSSQL backups. No need to elaborate og PG backups.

Reply:
I agree. Changed to:
"This command is not supported.  You have to backup the database using PostgreSQL techniques."